test/scip/testdata/field_inheritance.snapshot.rb

```diff
@@ -148,10 +153,13 @@ def self.get
 #           ^^^ definition [..] `<Class:D2>`#get().
      @@d2_x = @@d1_v + @@d1_x
 #    ^^^^^^ definition [..] `<Class:D2>`#`@@d2_x`.
-#             ^^^^^^ reference [..] `<Class:D1>`#`@@d1_v`.
-#                      ^^^^^^ reference [..] `<Class:D1>`#`@@d1_x`.
+#             ^^^^^^ reference [..] `<Class:D2>`#`@@d1_v`.
+#             relation definition=[..] `<Class:D1>`#`@@d1_v`.
+#                      ^^^^^^ reference [..] `<Class:D2>`#`@@d1_x`.
+#                      relation definition=[..] `<Class:D1>`#`@@d1_x`.
```

Comment:
Hmm, it is a little weird for the `definition=` to be emitted on this line instead of the definition line. If we do have a definition, then we should emit this at the definition instead of repeating it everywhere else.

Reply:
Ah, NVM, this makes sense, because the definition is `@@d2_x`  whereas this is `@@d1_x`